northd/northd.c

```diff
@@ -5735,6 +5735,12 @@ build_pre_acls(struct ovn_datapath *od, const struct hmap *port_groups,
                       "nd || nd_rs || nd_ra || mldv1 || mldv2 || "
                       "(udp && udp.src == 546 && udp.dst == 547)", "next;");
 
+        // Not to do conntrack on multicast packets.
```

Comment:
Please do not use `//` comments as requested in the [coding style document](https://github.com/ovn-org/ovn/blob/main/Documentation/internals/contributing/coding-style.rst#comments).

In general, a large part of the coding style requirements are checked by `checkpatch.py`, e.g.:
```
$ ./utilities/checkpatch.py -1
== Checking ab92e3e30746 ("do not send multicast packets to conntrack") ==
ERROR: C99 style comment
#19 FILE: northd/northd.c:5738:
        // Not to do conntrack on multicast packets.

Lines checked: 79, Warnings: 0, Errors: 1
```

Nit: I think I would use the same comment as in `build_pre_lb()`:
```
/* Do not send multicast packets to conntrack. */
```